icon/iiss/base.go

```diff
@@ -178,8 +179,42 @@ func handleConsensusInfo(wc state.WorldContext) error {
 		return err
 	}
 
-	// TODO update P-Rep status
+	// update P-rep status, vtotal, vfail, vfailcont
+	proposerExist := false
+	for _, p := range prepAddressList {
+		if p == proposer {
```

Comment:
Please use p.Equal() instead of simple "==" operator.

Reply:
I reflected it, Thanks.